src/components/popover-container/popover-container.stories.mdx

```diff
@@ -31,11 +32,10 @@ import PopoverContainer from "carbon-react/lib/components/popover-container";
 <Preview>
   <Story name="default" parameters={{ info: { disable: true }}} >
     <div style={{height: 100}}>
-      <PopoverContainer>
-      Contents
-      </PopoverContainer>
-         
-      </div>
+      <PopoverContainer p={8}>
```

Comment:
Not sure we want this padding on the default story do we?

Reply:
Ah yep forgot to remove